hayeon/programmers-2/최솟값-만들기.js

```diff
@@ -0,0 +1,39 @@
+function setSortedNumber(arr) {
+  return arr.sort((a, b) => a - b);
+}
+
+function solution(A, B) {
+  let sum = 0;
+
+  const sortedA = setSortedNumber(A);
+  const sortedB = setSortedNumber(B).reverse();
+
+  for (let i = 0; i < A.length; i++) {
+    sum += sortedA[i] * sortedB[i];
+  }
+
+  return sum;
+}
```

Comment:
```suggestion
function solution(A, B) {
  const sortedA = A.sort((a, b) => a - b);
  const sortedB = B.sort((a, b) => b - a);
 
  return sortedA.reduce((acc, cur, i) => acc + cur * sortedB[i], 0);
}
```
- 풀이잘봤습니다 🙇‍♂️
- 아래 주석으로 남겨주신 풀이처럼 이정도는 굳이 setSortedNumber 함수로 안뺴고 sort자체만으로 사용하셔도 괜찮습니다. reverse도 마찬가지입니다 ~!
- for문 말고 reduce를 활용하면 좀 더 깔끔한 풀이도 가능합니다 :)

Reply:
- 우와 reduce로도 풀 수 있겠군요!!! reduce가 여러모로 활용이 잘 되는 것 같네요.. 이제 풀 때 reduce로도 풀 수 있을까 생각하면서 풀어봐야겠습니당 감사합니다!!👍🏻